app/views/layouts/application.html.erb

```diff
@@ -4,6 +4,15 @@
   <title>Workshop-Portal</title>
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <script>
+    function enableDatepickers() {
+      $('[data-datepicker="true"]').datepicker({
+        language:"<%= I18n.locale %>",
+        format: "<%= I18n.t('date.formats.datepicker')%>",
```

Comment:
This should use the `escape_javascript` option: http://api.rubyonrails.org/classes/ActionView/Helpers/JavaScriptHelper.html#method-i-escape_javascript